builtin/mainmenu/content/dlg_package.lua

```diff
@@ -232,6 +233,53 @@ local function get_formspec(data)
 		else
 			table.insert_all(formspec, {"label[2,2;", fgettext("Loading..."), "]"} )
 		end
+	elseif current_tab == 4 then
+		if not package.releases and not data.releases_error and not data.releases_loading then
+			data.releases_loading = true
+
+			contentdb.get_package_releases(package, function(releases)
+				if not releases then
+					data.releases_error = true
+				end
+				ui.update()
+			end)
+		end
+
+		local function markup(text)
+			text = text:gsub("(%*%*)([^%s][^*]-[^%s])%*%*", "<b>%2</b>")
+			text = text:gsub("(%*)([^%s][^*]-[^%s])%*", "<i>%2</i>")
+			text = text:gsub("* ", "<b>•</b> ")
+			text = text:gsub("- ", "<b>•</b> ")
+			return text
+		end
+
+		local hypertext = ""
+		if package.releases then
+			for _, def in pairs(package.releases) do
+				hypertext = hypertext..
+					"<b>"..def.title.."</b> ("..def.release_date:gsub("T.*", "")..")\n"..
+					fgettext("$1 downloads",
+						def.downloads)..".\n"
+
+				if def["release_notes"] then
+					hypertext = hypertext..
+						markup((def["release_notes"]
+							:gsub("<!--.*", ""))
```

Comment:
Again this is broken, it will kill anything after the start of any HTML comment. As said you should use a proper API, like the one of CDB, to do the conversion.